### ISSUE TITLE
Meet with HQ for event set-up

### ISSUE THREAD

**Issue by jhibbets** [ashumz/citycamp#86]

**Comment by ChrisTheDBA** [comment]
Do we want Chad to attend this or have his dry run separate?

**Comment by dtraleigh** [comment]
I'll invite Chad when I have a time.

**Comment by jhibbets** [comment]
I was thinking this would be close to the event. I would like to get all
the presenations and have them ready to go. So I think this is seperate
from the walk-thru.

**Comment by dtraleigh** [comment]
Makes sense. I'll invite him anyway but emphasize that another walkthrough
a day or two before still needs to be done with final slides, etc.

**Comment by dtraleigh** [comment]
Meeting at HQ at 2pm, Wednesday June 3 with Chris and Tres for a walkthrough. We're meeting Allyson as well as two events interns.

**Comment by dtraleigh** [comment]
We met today at 2. Tres saw the space and all seemed ok. A few other to-do's will be generated from this but I'll close this issue now. I have uploaded the contract, which I signed, to the google drive.
